model/lifecycle_test.go

```diff
@@ -1038,71 +934,6 @@ func TestCreateBuildFromVersion(t *testing.T) {
 
 		})
 
-		Convey("the build should contain task caches that correspond exactly"+
```

Comment:
This test still seems useful, but just checking only tasks[i].Id

Reply:
I reinstated the test